src/App/App.js

```diff
@@ -3,14 +3,14 @@ import Header from '../Header/Header'
 import Main from '../Main/Main'
 import SingleView from '../SingleView/SingleView'
 import './App.css'
+import { Route, Switch } from 'react-router-dom'
 
 export class App extends Component {
   constructor() {
     super()
     this.state = {
       movies: null,
       filteredMovies: null,
-      singleMovie: null,
       isSingleView: false,
       isLoading: true,
```

Comment:
Great that you have some loading indication logic, and that you're using the standard pattern of setting to `true` to begin with.

Reply:
One thing that's currently being refactored is that I've gotten rid of all of the uses for `isSingleView` and have refactored to use the location prop on the render method, but that is not yet reflected in this PR

Reply:
That's great! I was going to say something about that but figured you were still working on Router stuff.